addon/index.js

```diff
@@ -0,0 +1,7 @@
+import ObjectController from './object_controller';
```

Comment:
i think we should drop this file,  people should just directly import the controller they need